src/ReactDOM.js

```diff
@@ -1,4 +1,3 @@
-/* global exports */
```

Comment:
I'm assuming this is just a comment and not some sort of pragma?

Reply:
It's used by linting tools, but our linting configuration already covers this.